modules/order/src/OrderRefreshInterface.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Drupal\commerce_order;
+
+use Drupal\commerce_order\Entity\OrderInterface;
+
+/**
+ * Service that runs refresh processes on an order.
```

Comment:
Everything's a service, no point in mentioning that. "Refreshes draft orders."? Should decide where we want to do the description of the full process, in the class docblock or the method docblock.